packages/common/src/HardhatConfig.js

```diff
@@ -61,6 +67,14 @@ function getHardhatConfig(configOverrides) {
     },
     namedAccounts: {
       deployer: 0
+    },
+    external: {
+      // This allows us to access the same deployed contract addresses that truffle can after running apply-registry,
+      deployments: {
```

Comment:
Adding this `deployments` section allows `deployments.get("X")` to read the address of `X` as written in `core/networks/NETWORK.json` after running `yarn apply-registry`. Apply registry sets addresses in `/core/build/contracts` which can now be read by hardhat.

Note that adding this deployments struct to the `path` object in `hardhat` doesn't work.

Reply:
nice. very useful that we can re-use these from all packages like this.